NEMO/models.py

```diff
@@ -2520,6 +2520,12 @@ def has_not_ended(self):
     def has_not_started(self):
         return False if self.start <= timezone.now() else True
 
+    def can_swap_user(self):
+        time_period = timedelta(hours=72)
+        if self.tool is not None and "TEM-" in self.tool.name:
```

Comment:
this seems to be the only equipment **hardcoded** string which is allowed for user swap; perhaps you can store this information in another table or update / migrate Models to add a boolean field to store this information?